community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -99,6 +100,7 @@
  */
 public class ImportTool
 {
+    private static final String UNLIMITED = "-";
```

Comment:
As @spacecowboy already mentioned that would be much more understandable to use `-1` as representation of unlimited value